diff --git a/setup.py b/setup.py
index 43dfc5c..cd5f30b 100644
--- a/setup.py
+++ b/setup.py
@@ -106,8 +106,7 @@ setup(
     # To provide executable scripts, use entry points in preference to the
     # "scripts" keyword. Entry points provide cross-platform support and allow
     # pip to create the appropriate form of executable for the target platform.
-    entry_points={'xicam.plugins.SettingsPlugin': ['cammart = xicam.gui.cammart:CamMartSettingsPlugin',
-                                                   'venvs = xicam.gui.cammart.venvs:VenvsSettingsPlugin'],
+    entry_points={'xicam.plugins.SettingsPlugin': [],
                   'xicam.plugins.CatalogPlugin': ['databroker_catalog_plugin = xicam.gui.bluesky.databroker_catalog_plugin:DatabrokerCatalogPlugin']},
     ext_modules=[],
     include_package_data=True,
